client/src/common/components/Home.tsx

```diff
@@ -1,8 +1,149 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+
+import {
+	Accordion,
+	AccordionItem,
+	AccordionItemHeading,
+	AccordionItemButton,
+	AccordionItemPanel,
+} from 'react-accessible-accordion';
+import 'react-accessible-accordion/dist/fancy-example.css';
 
 const Home = () => {
 	// test, added style here just to see the classes work
-	return <div　className="bg-primary_bg_color w-screen h-96 mt-headerHeight text-primary_title_color">This is Home</div>;
+	return (
+		<div className="bg-primary_bg_color text-primary_title_color bg-gradient">
+			<div className="p-10 md:p-20">
+				<div className="md:float-right w:20 md:w-2/4 ">
+					<img src="./assets/shared_workspace.svg" />
```

Comment:
@Dannyboy222 Please add an `alt` attribute to every `img` element.

```suggestion
					<img className="md:float-right w:20 md:w-2/4"  src="./assets/shared_workspace.svg" />
```
^ doen't this work?